### PR TITLE
Add generate-workflows usage to CONTRIBUTING

### PR DESCRIPTION
**Pull request by tammy-baylis-swi**
# Description

Adds missing info on generate_workflows.py usage.

Partially fixes https://github.com/open-telemetry/opentelemetry-python/issues/4263

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

n/a

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [X] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [X] Documentation has been updated

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -306,6 +307,6 @@ When adding support for a new Python release remember to:
 
 - Add the version in `tox.ini`
 - Add the version in `pyproject.toml` trove classifiers
-- Update github workflows accordingly; lint and benchmarks use the latest supported version
+- Update github workflows accordingly with `tox -e generate-workflows`; lint and benchmarks use the latest supported version
```
Comment (xrmx) [inline CONTRIBUTING.md:310]: This is valid for the `Bumping the Python baseline` chapter too

Comment (tammy-baylis-swi) [reply CONTRIBUTING.md:311]: Added in [b3e6e84](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2977/commits/b3e6e84e596dafb32c13255cb5b7da12a8a56e87)

**Comment by tammy-baylis-swi** [comment]
Do we want an Issue and Changelog update for this?

**Review by emdneto — APPROVED** [review]
Thanks! Could you please do the same for core? Maybe you can reference this issue: https://github.com/open-telemetry/opentelemetry-python/issues/4263

**Comment by tammy-baylis-swi** [comment]
> Thanks! Could you please do the same for core? Maybe you can reference this issue: [open-telemetry/opentelemetry-python#4263](https://github.com/open-telemetry/opentelemetry-python/issues/4263)

Sounds good! Core PR here: https://github.com/open-telemetry/opentelemetry-python/pull/4266

**Review by xrmx — APPROVED** [review]

**Pull request merged**